repos/system_upgrade/el7toel8/actors/nssglibccheck/tests/test_nss_glibc.py

```diff
@@ -0,0 +1,33 @@
+from leapp.snactor.fixture import current_actor_context
+
+TEST_DATA = '''
+#     passwd: sss files wins # from profile
+#     hosts: files dns  # from user file
+
+passwd:     sss files systemd
+group:      sss files systemd
+netgroup:   sss files wins
+automount:  sss files winbind
+services:   sss files
+foo:        sss files #winbind
+'''
+
+
+def test_blacklister(current_actor_context):
+    caught = list(current_actor_context._check_modules(
```

Comment:
this is not an instance of an actor
One cannot get an instance of an actor. This is why we mostly put the code into actor libraries and test the libraries instead. The actor itself becomes a simple def process(self): library.process() most of the times.

Reply:
Ahh, yeah totally forgot about that, thanks!